test/integration/test-actions.js

```diff
@@ -17,81 +17,73 @@
 import {AmpEvents} from '../../src/amp-events';
 import {createFixtureIframe, poll} from '../../testing/iframe.js';
 
-describe
-  .configure()
-  .retryOnSaucelabs()
-  .run('on="..."', () => {
-    let fixture;
-
-    beforeEach(() => {
-      return createFixtureIframe('test/fixtures/actions.html', 500).then(
-        (f) => {
-          fixture = f;
-
-          // Wait for one <amp-img> element to load.
-          return fixture.awaitEvent(AmpEvents.LOAD_END, 1);
-        }
-      );
+describe.configure().run('on="..."', () => {
```

Comment:
Can be simplified to `describe('... `

Reply:
Yes, it certainly can. Good catch, here and elsewhere!